.vscode/settings.json

```diff
@@ -1,3 +1,4 @@
 {
-    "typescript.tsdk": "node_modules/typescript/lib"
+    "typescript.tsdk": "node_modules/typescript/lib",
+    "editor.formatOnSave": true
```

Comment:
add .vscode/settings to your global git ignore

Reply:
We should not force other people to formatOnSave later they call polis.

Reply:
unless there are very specific thing (like build stuff), should not push anything related to idea at all ...

Reply:
sorry my bad, it slipped through and made into this commit. I'll remove it